miden-store/src/server/api.rs

```diff
@@ -0,0 +1,81 @@
+use std::net::ToSocketAddrs;
+use tracing::info;
+use anyhow::Result;
+
+use crate::config::StoreConfig;
+use tonic::{transport::Server, Request, Response, Status, Streaming};
+
+use miden_proto::store;
+
+#[derive(Default)]
+pub struct DBApi {}
+
+#[tonic::async_trait]
+impl store::api_server::Api for DBApi {
+    type CheckNullifiersStream = Streaming<store::CheckNullifiersResponse>;
+    type ListNotesByTagStream = Streaming<store::ListNotesByTagResponse>;
+
+    async fn get_block_header(
+        &self,
+        _request: Request<store::GetBlockHeaderRequest>,
+    ) -> Result<Response<store::GetBlockHeaderResponse>, Status>
+    {
+        todo!()
+    }
+
+    async fn get_note(
+        &self,
+        _request: Request<store::GetNoteRequest>,
+    ) -> Result<Response<store::GetNoteResponse>, Status>
+    {
+        todo!()
+    }
+
+    async fn check_nullifiers(
+        &self,
+        _request: tonic::Request<Streaming<store::CheckNullifiersRequest>>,
+    ) -> Result<Response<Self::CheckNullifiersStream>, Status>
+    {
+        todo!()
+    }
+
+    async fn get_transaction_data(
+        &self,
+        _request: Request<store::GetTransactionDataRequest>,
+    ) -> Result<Response<store::ResponseBytes>, Status>
+    {
+        todo!()
+    }
+
+    async fn list_notes_by_tag(
+        &self,
+        _request: Request<store::ListNotesByTagRequest>,
+    ) -> Result<Response<Self::ListNotesByTagStream>, Status>
+    {
+        todo!()
+    }
+
+    async fn submit_transaction(
+        &self,
+        _request: Request<store::SubmitTransactionRequest>,
+    ) -> Result<Response<store::ResponseBytes>, Status>
+    {
+        todo!()
+    }
```

Comment:
As mentioned in some other comments - this is not relevant here, so needs to be removed.

Reply:
removed